internal/services/v1/experimental_test.go

```diff
@@ -638,17 +637,32 @@ func TestBulkCheckPermission(t *testing.T) {
 
 			expected := make([]*v1.BulkCheckPermissionPair, 0, len(tt.response))
 			for _, r := range tt.response {
-				reqRel := tuple.ParseRel(r.req)
+				reqRel := tuple.MustParse(r.req)
 				resp := &v1.BulkCheckPermissionPair_Item{
 					Item: &v1.BulkCheckPermissionResponseItem{
 						Permissionship: r.resp,
 					},
 				}
+
+				rel := reqRel.Subject.Relation
+				if rel == tuple.Ellipsis {
+					rel = ""
+				}
```

Comment:
don't we have a helper function for this?

Reply:
Sort of: we can use `stringz.Default`

Reply:
Changed